.travis.yml

```diff
@@ -124,5 +124,9 @@ script:
   # Texas Instruments dra7xx
   - PLATFORM=ti  PLATFORM_FLAVOR=dra7xx  make -j8 all -s
 
+  # FSL ls1021a
+  - PLATFORM=ls  PLATFORM_FLAVOR=ls1021atwr CROSS_COMPILE=arm-linux-gnueabihf- make -j8 all -s
+  - PLATFORM=ls  PLATFORM_FLAVOR=ls1021aqds CROSS_COMPILE=arm-linux-gnueabihf- make -j8 all -s
```

Comment:
Please do not override CROSS_COMPILE, it is already set above to `ccache arm-linux-gnueabihf-`